src/rbd.cc

```diff
@@ -2871,7 +2877,7 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
     cout << tbl;
   }
 
-  return 0;
+  return r;
```

Comment:
NIT: I'd feel better if this said return r < 0 ? r : 0;

In the current code it doesn't make any difference, but I'd like to be guaranteed that this returns 0 on success as before.